mock/py/mockbuild/backend.py

```diff
@@ -720,6 +721,7 @@ def rebuild_package(self, spec_path, timeout, check, dynamic_buildrequires):
         # --nodeps because rpm in the root may not be able to read rpmdb
         # created by rpm that created it (outside of chroot)
         check_opt = []
+        depsonly = self.config["calculatedeps"]
```

Comment:
depsonly vs. calculatedeps confuses me when reviewing this PR. I would use the same name. I.e. s/depsonly/calculatedeps/.